org.sailfishos.usermanager.xml

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN"
+"http://www.freedesktop.org/standards/dbus/1.0/introspect.dtd">
+<node>
+  <interface name="org.sailfishos.usermanager">
+    <method name="users">
+        <arg direction="in" type="(i)" name="message"/>
```

Comment:
Hmm... What kind of message is that and why is it array of 32-bit ints?

Reply:
It is actually a struct containing one int, but looks like some kind of qdbusxml2cpp wizardry.

Reply:
Annotation says it's QDBusMessage but I don't think that's how they should be used.

Reply:
> It is actually a struct containing one int, but looks like some kind of qdbusxml2cpp wizardry.

Your right, struct. That doesn't make it less weird, though.

Reply:
It's not documented anywhere but qdbusxml2cpp requires those brackets for some reason, the type inside does not matter as it gets replaced.

Reply:
It's not good when such an obscure implementation detail finds its way into the interface. The whole purpose of an interface is to abstract out implementation details, isn't it?